test/specs/mock/delete-collection.spec.js

```diff
@@ -543,14 +543,14 @@ for (let spec of specs) {
                         Photo: {
                           fieldname: "Photo",
                           originalname: "1MB.jpg",
+                          destination: res2.body[0].Photo.destination,
+                          filename: res2.body[0].Photo.filename,
                           name: res2.body[0].Photo.name,
                           encoding: "7bit",
                           mimetype: "image/jpeg",
                           path: res2.body[0].Photo.path,
                           extension: "jpg",
                           size: 683709,
-                          truncated: false,
-                          buffer: null
```

Comment:
This file is a good example of the changes. I believe if `memoryStorage` is used you will actually get a buffer here, and possibly `truncated`, though I believe `stream` is the encouraged access point regardless. Also notable that stream is `defineProperty` so it doesn't show up when serialized.

Reply:
This is fine.  But please call it out in the changelog for 4.0.0.